lingua_franca/lang/parse_en.py

```diff
@@ -1685,3 +1710,256 @@ def numbers_to_digits(self, utterance):
 def normalize_en(text, remove_articles=True):
     """ English string normalization """
     return EnglishNormalizer().normalize(text, remove_articles)
+
+
+def extract_timespan_en(text, time_unit=TimespanUnit.TIMEDELTA, replace_token=""):
+    """
+    Convert an english phrase into a number of seconds
```

Comment:
Isn't this just `extract_duration`?

Reply:
yes but it changes return type and adds the replace token, its mainly a compatibility thing so extract_duration always means the same (mycroft) thing